baseline_configs/one_phase/procthor/ithor/ithor_rearrange_base.py

```diff
@@ -0,0 +1,486 @@
+import copy
```

Comment:
File seems essentially identical to
```
baseline_configs/rearrange_base.py
```

Reply:
Yes, same as with [ithor_one_phase_rgb_base.py](https://github.com/allenai/ai2thor-rearrangement/pull/39/files/41d3f8497fc53f47857e468ff8b9aaec0608df0c#diff-1bc0250466b4fa74b10ad351a89b812fbba5e6ce6bb604cdb4f30df04816f72c) above.